spec/support/unit/shared_examples/set_session_or_flash.rb

```diff
@@ -56,9 +56,28 @@
 
     context 'in the positive' do
       context 'if the given key is present in the store' do
-        it 'accepts' do
-          controller = controller_with_store('the key' => 'any value')
-          expect(controller).to set_store['the key']
+        context 'and controller set the key as a string' do
+          it 'accepts the param as a string' do
+            controller = controller_with_store('the_key' => 'any value')
+            expect(controller).to set_store['the_key']
+          end
+
+          it 'accepts the param as a symbol' do
+            controller = controller_with_store('the_key' => 'any value')
+            expect(controller).to set_store[:the_key]
+          end
+        end
+
+        context 'and controller set the key as a symbol' do
```

Comment:
Do we need these tests? I think that `controller_with_store(the_key: 'any value')` will translate to `flash[:the_key] = 'any value'` which should work regardless of what is given to the matcher. This seems like testing Rails.